Update multi CSI driver related docs (#399)
harvester/docs: docs/advanced/settings.md

_Available as of v1.2.0_

> If you install third-party CSI drivers in the Harvester cluster, you need to configure some necessary informations through this setting before using **Backup/Snapshot** related features.

Reviewer: ```suggestion
If you install third-party CSI drivers in the Harvester cluster, you must configure some necessary information through this setting before using **Backup/Snapshot** related features.
```
